src/main/java/org/mockbukkit/mockbukkit/inventory/ItemStackMock.java

```diff
@@ -282,7 +356,7 @@ public boolean equals(Object obj)
 		}
 		if (stack instanceof ItemStackMock bukkit)
 		{
-			return isSimilar(bukkit) && this.amount == bukkit.getAmount() && this.durability == bukkit.durability && Objects.equals(this.getLore(), bukkit.getLore()) && Objects.equals(this.getEnchantments(), bukkit.getEnchantments());
+			return isSimilar(bukkit) && this.amount == bukkit.getAmount() && this.durability == bukkit.durability && Objects.equals(this.itemMeta, bukkit.getItemMeta());
```

Comment:
Does this match Paper's impl? I remember it being weird and not actually checking if *everything* is equal, not sure if they changed that at some point.

Reply:
I checked this, and it seems to check everything now

Reply:
Thanks for checking !